src/Lykke.Job.TradeDataAggregator.Services/TradeDataAggregationService.cs

```diff
@@ -41,13 +43,13 @@ public static string CreateKey(string limitOrder, DateTime dt)
         private readonly Dictionary<string, TemporaryAggregatedData> _tempDataByLimitOrderAndDtId = new Dictionary<string, TemporaryAggregatedData>();
 
         public TradeDataAggregationService(
-            IClientTradesRepository clientTradesRepository,
+            IOperationsHistoryClient operationsHistoryClient,
             IMarketDataRepository marketDataRepository,
             IAssetsServiceWithCache assetsService,
             IAssetPairBestPriceRepository assetPairBestPriceRepository,
             ILog log)
         {
-            _clientTradesRepository = clientTradesRepository;
+            _operationsHistoryClient = operationsHistoryClient;
```

Comment:
I would check ctor parameters for null explicitly or by [NotNull] attribute. It can save debugging time.